docker/common-services.yaml

```
@@ -150,3 +150,34 @@ services:
       # https_proxy: ${HTTPS_PROXY_URL}
       # no_proxy: ${NO_PROXY_LIST}
     restart: always
+
+  teams-delegated-operator-worker-common:
+    image: voxel51/fiftyone-app:v2.1.0
+    deploy:
+      replicas: ${FIFTYONE_DELEGATED_OPERATOR_WORKER_REPLICAS:-0}
+    environment:
+      API_URL: ${API_URL}
+      FIFTYONE_API_KEY: ${FIFTYONE_API_KEY}
+      FIFTYONE_AUTH_SECRET: ${FIFTYONE_AUTH_SECRET}
+      FIFTYONE_DATABASE_ADMIN: false
+      FIFTYONE_DATABASE_NAME: ${FIFTYONE_DATABASE_NAME}
+      FIFTYONE_DATABASE_URI: ${FIFTYONE_DATABASE_URI}
+      FIFTYONE_DEFAULT_APP_ADDRESS: 0.0.0.0
+      FIFTYONE_DEFAULT_APP_PORT: 5151
+      FIFTYONE_ENCRYPTION_KEY: ${FIFTYONE_ENCRYPTION_KEY}
+      FIFTYONE_INTERNAL_SERVICE: true
+      FIFTYONE_MEDIA_CACHE_APP_IMAGES: false
+      FIFTYONE_MEDIA_CACHE_SIZE_BYTES: -1
+      FIFTYONE_PLUGINS_CACHE_ENABLED: true
+      FIFTYONE_PLUGINS_DIR: /opt/plugins
```

some of these are not needed.
```suggestion
      API_URL: ${API_URL}
      FIFTYONE_API_KEY: ${FIFTYONE_API_KEY}
      FIFTYONE_DATABASE_ADMIN: false
      FIFTYONE_DATABASE_NAME: ${FIFTYONE_DATABASE_NAME}
      FIFTYONE_DATABASE_URI: ${FIFTYONE_DATABASE_URI}
      FIFTYONE_ENCRYPTION_KEY: ${FIFTYONE_ENCRYPTION_KEY}
      FIFTYONE_INTERNAL_SERVICE: true
      FIFTYONE_MEDIA_CACHE_SIZE_BYTES: -1
      FIFTYONE_PLUGINS_CACHE_ENABLED: true
      FIFTYONE_PLUGINS_DIR: /opt/plugins
```

---

does this never talk to CAS?  you need the Auth Secret to authenticate with CAS...

---

Nope it talks to the API which may talk to CAS.

---

a'ight then